.github/workflows/azure_e2e_test.yml

```diff
@@ -1,27 +1,19 @@
 name: 'Azure E2E test'
-on:
+'on':
+  schedule:
+    # run twice a day at 10:00 and 22:00 UTC every day of the week
+    - cron: "0 10/12 * * 1-5"
   workflow_dispatch:
     inputs:
       target-branch:
         description: 'Branch of CBAS to run tests on'
         required: true
```

Comment:
Does this still need to be required if there is a default value provided? Also, CBAS has a `main` branch not a `develop` branch

Reply:
Oops this should definitely be main. I haven't been using this in testing but my though behind this was that if in future we want to run this test against PRs passing a branch would be easier. But on second thought I guess when we get to it there would be other changes needed to make that work so I guess I can remove that for now. 🤔